src/ElementScreenTaker.js

```diff
@@ -4,55 +4,71 @@ let captureWebsite = require('capture-website');
 let puppeteer = require('puppeteer');
 let cheerio = require('cheerio');
 let id = [];
-let url = process.cwd()+`\\src\\test.html`;
+let url = process.cwd() + `\\src\\test.html`;
 
 export default class ElementScreenTaker {
     constructor(value) {
-
     }
 
 
     // takes a screen shot from each newsarticle on the generated html page'
     //test: returns image Path
     //aantal plaatjes maken via varible
     async takeScreenShots() {
-        try{
-            await this.getCssSelectors();
+        try {
+            await this.getCssSelectors(process.env.selector);
             let time = new Date();
-            let t = time.getDate()+"_"+time.getMonth()+"_"+time.getFullYear()+"_"+time.getHours()+"_"+time.getMinutes();
+            let t = time.getDate() + "_" + time.getMonth() + "_" + time.getFullYear() + "_" + time.getHours() + "_" + time.getMinutes();
```

Comment:
ik zou hier ook de javascript string interpolation voor gebruiken.

vb: ``` `${time.getDate()}_${time.getMonth()}_${time.getHours()}_${time.getMinutes()}` ```

een andere mogelijkheid is om de package dateformat te gebruiken:
https://www.npmjs.com/package/dateformat

op die manier kan je dan een custom string laten generegen als volgt:
```
var dateFormat = require('dateformat');
var now = new Date();
dateFormat(now, "dd_MM_yyyy_HH_mm");
```